open_spiel/games/quoridor.h

```diff
@@ -28,20 +28,25 @@
 //   "board_size"        int     Size of the board (default = 9)
 //   "wall_count"        int     How many walls per side (default = size^2/8)
 //   "ansi_color_output" bool    Whether to color the output for a terminal.
+//   "players"           int     Number of players (default = 2)
 
 namespace open_spiel {
 namespace quoridor {
 
-inline constexpr int kNumPlayers = 2;
+inline constexpr int kDefaultNumPlayers = 2;
+inline constexpr int kMinNumPlayers = 2;
+inline constexpr int kMaxNumPlayers = 4;
 inline constexpr int kDefaultBoardSize = 9;
 inline constexpr int kMinBoardSize = 3;
 inline constexpr int kMaxBoardSize = 25;
 inline constexpr int kMaxGameLengthFactor = 4;
-inline constexpr int kCellStates = 1 + kNumPlayers;
+inline constexpr int kCellStates = 1 + kDefaultNumPlayers;
```

Comment:
I am not sure what this value should be. 
I think it must be adapted according to `num_players_`.
Maybe, the initial author knows better, @tewalds.

Reply:
Yes this should no longer be a constant. You should remove it and instead make it a function inside the game or state object and call that function whenever that value is need. (Kind of surprised the 4-player case would work at all without this change)